src/logic/__tests__/validateCurrentProject.test.js

```diff
@@ -0,0 +1,56 @@
+import validateCurrentProject from '../validateCurrentProject';
+
+import Analyzer from '../../analyzers';
+import validateSource from '../helpers/validateSource';
+import {getCurrentProject} from '../../selectors';
+
+const mockHtmlSource = '<html></html>';
+const mockCssSource = 'div {}';
+
+jest.mock('../../analyzers');
+jest.mock('../helpers/validateSource');
+jest.mock('../../selectors', () => ({
+  getCurrentProject: jest.fn(() => ({
+    sources: {
+      html: mockHtmlSource,
+      css: mockCssSource,
+    },
+  })),
+}));
```

Comment:
To be honest this also intuitively is probably not something I would have mocked if I were writing this test, although I think the argument is _much_ less clear-cut than with the helper module commented on above. I could truly go either way on this, but wanted to flag it as something to think about, particularly in terms of “which decision is going to make this test more robust?”, with “robust” meaning “sensitive to breaking changes in the code under test, and insensitive to non-breaking changes in the code under test”.